src/app.component.scss

```diff
@@ -0,0 +1,77 @@
+@import "./utils/constants";
+
+:host {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  height: 100vh;
+  background-color: $color-background;
+
+  .header {
```

Comment:
Is it because of the concern of possibly duplicating properties?

I personally like to group them with similar properties (dimensions, flex-related, text-related, etc.) since modern IDEs warn the duplicates. I feel like `width` not with `height`, or `align-items` not with `justify-content` would look a bit off.